apps/src/lab2/progress/ProgressManager.ts

```diff
@@ -75,6 +76,12 @@ export default class ProgressManager {
 
     // Go through each validation to see if we have a match.
     for (const validation of this.currentValidations) {
+      // If it's not a next validation, then make sure the lab-specific validator
```

Comment:
nit: might not be obvious what "next" means in this context, maybe could add something like `// If it's not a passing validation (i.e. validation.next is false), then...`

Reply:
Good call.  Updated.  I do wonder whether we could rename `next` to `success` everywhere, in a bigger follow-up.